fastlane/Fastfile

```diff
@@ -64,26 +62,52 @@ platform :ios do
     clean_build_artifacts
 
   end
+  
+  desc "New release to testflight for external users."
+  desc "Parameters:"
+  desc "- bump_type (optional): represents the type of deploy. If not specified, the user will be asked for it." 
+  lane :release_testflight do |options|
+    release build_configuration: :testflight, bump_type: options[:bump_type]
+  end
+
+  desc "New release to testflight for QA(Alpha). This lane will never update the testflight version, only the build number."
```

Comment:
Same TestFlight. 

What do you mean by "the testflight version"? I'd just say the version.